tests/test_user.py

```diff
@@ -127,17 +130,19 @@ def test_edit_as_superuser(
         cmd = f"{cmd} --make-superuser"
     if new_is_non_verified:
         cmd = f"{cmd} --remove-verified"
-    debug(cmd)
+
     res = invoke_as_superuser(cmd)
-    debug(res.data)
-    assert res.retcode == 0
-    assert res.data["email"] == NEW_EMAIL
-    assert res.data["cache_dir"] == NEW_CACHE_DIR
-    assert res.data["slurm_user"] == NEW_SLURM_USER
-    assert res.data["username"] == NEW_USERNAME
-    assert res.data["is_superuser"] == new_is_superuser
+
     if new_is_non_verified:
+        assert res.retcode == 0
+        assert res.data["email"] == NEW_EMAIL
+        assert res.data["cache_dir"] == NEW_CACHE_DIR
+        assert res.data["slurm_user"] == NEW_SLURM_USER
+        assert res.data["username"] == NEW_USERNAME
+        assert res.data["is_superuser"] == new_is_superuser
         assert not res.data["is_verified"]
+    else:
+        assert res.retcode == 1
```

Comment:
If we add another assertion for `res.data`, this branch becomes a bit more readable.